[32m2021-06-28 15:40:29[0m - dagster - [34mDEBUG[0m - catalog_pipeline - 45b977e8-de2e-4614-9f3e-88b257245366 - 1896 - creators_list - LOGS_CAPTURED - Started capturing logs for solid: creators_list.
[32m2021-06-28 15:40:29[0m - dagster - [34mDEBUG[0m - catalog_pipeline - 45b977e8-de2e-4614-9f3e-88b257245366 - 1896 - creators_list - STEP_START - Started execution of step "creators_list".
[32m2021-06-28 15:40:30[0m - dagster - [34mDEBUG[0m - catalog_pipeline - 45b977e8-de2e-4614-9f3e-88b257245366 - 1896 - creators_list - LOADED_INPUT - Loaded input "df" using input manager "io_manager", from output "result" of step "extract_dimensions"
[32m2021-06-28 15:40:30[0m - dagster - [34mDEBUG[0m - catalog_pipeline - 45b977e8-de2e-4614-9f3e-88b257245366 - 1896 - creators_list - STEP_INPUT - Got input "df" of type "Any". (Type check passed).
[32m2021-06-28 15:40:30[0m - dagster - [34mERROR[0m - [31mcatalog_pipeline - 45b977e8-de2e-4614-9f3e-88b257245366 - 1896 - creators_list - STEP_FAILURE - Execution of step "creators_list" failed.

dagster.core.errors.DagsterExecutionStepExecutionError: Error occurred while executing solid "creators_list"::

IndexError: Too many levels: Index has only 1 level, not 2

Stack Trace:
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/dagster/core/execution/plan/utils.py", line 42, in solid_execution_error_boundary
    yield
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/dagster/utils/__init__.py", line 382, in iterate_with_context
    next_output = next(iterator)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/dagster/core/definitions/decorators/solid.py", line 307, in _coerce_solid_compute_fn_to_iterator
    result = fn(context, **kwargs) if context_arg_provided else fn(**kwargs)
  File "/mnt/d/ims/situated-views/bin/solids/catalog.py", line 118, in creators_list
    listed_creators.droplevel(1)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/pandas/core/generic.py", line 764, in droplevel
    new_labels = labels.droplevel(level)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 1609, in droplevel
    levnums = sorted(self._get_level_number(lev) for lev in level)[::-1]
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 1609, in <genexpr>
    levnums = sorted(self._get_level_number(lev) for lev in level)[::-1]
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 1484, in _get_level_number
    self._validate_index_level(level)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 1475, in _validate_index_level
    raise IndexError(
[0m
